app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/paypal/PaypalScreen.kt

```diff
@@ -0,0 +1,528 @@
+        }
+        PaypalLoadingScreen()
+        LaunchedEffect(key1 = Unit) {
+          launcher.launch(viewModelState.url)
+        }
+      }
+
+      is PaypalUIState.BillingAgreementAvailable ->
+        BillingAgreementScreen(
+          buyingPackage = viewModelState.purchaseRequest.domain,
+          onBuyClick = { viewModelState.onBuyClick() },
+          onOtherPaymentMethodsClick = onOtherPaymentMethodsClick,
+          onRemoveBillingAgreementClick = viewModelState.onRemoveBillingAgreementClick,
+          paymentMethodName = viewModelState.paymentMethodName,
+          paymentMethodIconUrl = viewModelState.paymentMethodIconUrl,
+        )
+
+      PaypalUIState.PaypalAgreementRemoved -> {
+        PaypalLoadingScreen()
+        onOtherPaymentMethodsClick()
+      }
+    }
+  }
+}
+
+@Composable
+private fun PaypalErrorScreen(
+  onRetryClick: () -> Unit,
+  onContactUs: () -> Unit,
+) {
+  when (LocalConfiguration.current.orientation) {
+    Configuration.ORIENTATION_LANDSCAPE ->
+      LandscapePaymentErrorView(
+        onRetryClick = onRetryClick,
+        onContactUsClick = onContactUs
+      )
+
+    else -> PortraitPaymentErrorView(
+      onRetryClick = onRetryClick,
+      onContactUsClick = onContactUs
+    )
+  }
+}
+
+@Composable
+private fun PayPalNoConnectionScreen(onRetryClick: () -> Unit) {
+  when (LocalConfiguration.current.orientation) {
+    Configuration.ORIENTATION_LANDSCAPE ->
+      LandscapePaymentsNoConnectionView(onRetryClick = onRetryClick)
+
+    else -> PortraitPaymentsNoConnectionView(onRetryClick = onRetryClick)
+  }
+}
+
+@Composable
+private fun PaypalAccountScreen(
+  modifier: Modifier = Modifier,
+  paymentMethodName: String,
+  paymentMethodIconUrl: String,
+  onRemoveBillingAgreementClick: () -> Unit,
+) {
+  Column(
+    modifier = modifier.padding(vertical = 8.dp),
+    horizontalAlignment = Alignment.End
+  ) {
+    Row(
+      modifier = Modifier
+        .fillMaxWidth()
+        .background(greyLight)
+        .border(1.dp, agBlack)
+        .padding(16.dp),
+      verticalAlignment = Alignment.CenterVertically,
+    ) {
+      AptoideAsyncImage(
+        modifier = Modifier.size(22.dp),
+        data = paymentMethodIconUrl,
+        contentDescription = null
+      )
+
+      Text(
+        modifier = Modifier
+          .fillMaxWidth()
+          .weight(1f)
+          .padding(horizontal = 16.dp),
+        text = paymentMethodName,
+        maxLines = 2,
+        style = AppTheme.typography.descriptionGames,
+        overflow = TextOverflow.Ellipsis,
+        color = agBlack
+      )
+      Image(
+        imageVector = getCheck(agBlack),
+        contentDescription = null,
+      )
+    }
+    Row(
+      modifier = Modifier
+        .clickable(onClick = onRemoveBillingAgreementClick)
+        .padding(start = 8.dp, top = 8.dp, bottom = 8.dp),
+      verticalAlignment = Alignment.CenterVertically
+    ) {
+      Text(
+        modifier = Modifier.padding(end = 8.dp),
```

Comment:
This padding is different from design. It is 16 in design

Reply:
fixed